LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Stefan Gasser <stefangasser.info@gmail.com> and Mike North <michael.l.north@gmail.com>
+Copyright (c) 2016 Stefan Gasser <stefangasser.info@gmail.com> and Mike North <michael.l.north@gmail.com>
 
 Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
 
```

Comment:
Probably a bad rebase. You may want to create a new branch and copy your changes over